File: HOL/java/modern-cloud-apps/end/src/main/java/devCamp/WebApp/Utils/IncidentApiHelper.java
package devCamp.WebApp.Utils;

import java.net.URI;
import java.net.URISyntaxException;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.stereotype.Component;

import devCamp.WebApp.IncidentAPIClient.IncidentAPIClient;

public class IncidentApiHelper {
	public static IncidentAPIClient getIncidentAPIClient() {
		
		String apiurl= System.getenv("INCIDENT_API_URL");
		IncidentAPIClient client = null;
		
		String uri = String.format("https://%s",apiurl);
		client = new IncidentAPIClient(uri,"");
		return client;
	}
}

File: HOL/dotnet/02-modern-cloud-apps/end/DevCamp.WebApp/Utils/IncidentApiHelper.cs
using IncidentAPI;
using System;

namespace DevCamp.WebApp.Utils
{
    public class IncidentApiHelper
    {
        public static IncidentAPIClient GetIncidentAPIClient()
        {
            var client = new IncidentAPIClient(new Uri(Settings.INCIDENT_API_URL));
            return client;
        }
    }
}
